packages/actionmenu/src/react/overlay.js

```diff
@@ -11,21 +11,31 @@ const styles = {
   overlay: () => css(stylesheet['.psds-actionmenu__overlay'])
 }
 
+const hasComponentRendered = () =>
```

Comment:
Will this break SSR?  window is not defined outside the browser.

Reply:
Right, so ideally line 16 catches that?

Reply:
This is the same format done in the position package

Reply:
https://github.com/JedWatson/exenv is the react core way to do this check

Reply:
I have never seen this. It's a package last updated almost three years ago. 

You want me to update the window check with this package?